.github/workflows/docs.yml

```diff
@@ -71,12 +71,17 @@ jobs:
         ssh-keyscan github.com >> ~/.ssh/known_hosts
         
         # Checkout allennlp-docs to /allennlp-docs
-        mkdir -p /ssh
-        echo $DOCS_DEPLOY_KEY > /ssh/allennlp-docs.key
-        ssh-add /ssh/allennlp-docs.key
-        GIT_SSH_COMMAND='ssh -i /ssh/allennlp-docs.key' git clone git@github.com:allenai/allennlp-docs.git /allennlp-docs
+        echo $DOCS_DEPLOY_KEY > ~/allennlp-docs.key
```

Comment:
Would have had to add sudo to a bunch of these commands if we didn't put the key in the home directory

Reply:
Do you have to set the permissions on the key to `600`? Sometimes `ssh` is finicky about that.